%YAML:1.0
minDistBetweenBlobs: 5.0
minThreshold: 80
maxThreshold: 255
thresholdStep: 10
filterByInertia: 0
minInertiaRatio: 0.0
maxInertiaRatio: 1.0
filterByConvexity: 0
minConvexity: 0.0
maxConvexity: 1.0
filterByColor: 1
blobColor: 255
filterByCircularity: 0
minCircularity: 0.0
maxCircularity: 1.0
filterByArea: 1
minArea: 1.0
maxArea: 50.0
